src/OpenAL/Extensions/Silk.NET.OpenAL.Extensions.Creative/EnumerateAll.cs

```diff
@@ -43,7 +43,7 @@ public IEnumerable<string> GetStringList(GetEnumerateAllContextStringList param)
                 var currentPos = result;
                 while (true) {
                     var currentString = Marshal.PtrToStringAnsi(new IntPtr(currentPos));
-                    if (currentString is null) {
+                    if (!string.IsNullOrEmpty(currentString)) {
```

Comment:
The logic is inversed now. I think you have to remove the negation.

Reply:
Ah thanks!

Reply:
Fixed.